docs/content/connectors/oceanbase-cdc.md

```diff
@@ -314,6 +314,13 @@ The OceanBase CDC Connector contains some options for both sql and stream api as
                 <td>String</td>
                 <td>Option to pass custom JDBC URL properties. User can pass custom properties like 'jdbc.properties.useSSL' = 'false'.</td>
             </tr>
+            <tr>
+                <td>obcdc.properties.*</td>
+                <td>optional</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Option to pass custom configurations to the <code>libobcdc</code>, eg: 'obcdc.properties.sort_trans_participants' = '1'.</td>
```

Comment:
can we add doc page [libobcdc configs](https://www.oceanbase.com/docs/common-oceanbase-database-cn-1000000000221095) to this description to guide user what configurations are avaidable.

Reply:
Sure, thanks for reminding.